config/peerpods/podvm/gcp-podvm-image-handler.sh

```diff
@@ -188,6 +202,44 @@ function create_image_from_prebuilt_artifact() {
   echo "GCP image created successfully from prebuilt artifact"
 }
 
+function image_exists() {
+    echo "Checking if GCP image exists"
+
+    local image_path latest_image_id
+
+    # This command returns something like this:
+    # https://www.googleapis.com/compute/v1/projects/my-project/global/images/my-image
+    # Use `cut` to the image name
```

Comment:
It is obvious that `cut` is used below, it is less obvious what we want to drop.
```suggestion
    # we want to drop the leading "https://www.googleapis.com/compute/v1/"
```